### PR TITLE
CHORE: add html,assets and style.css

### PR DESCRIPTION
**Pull request by 29td**
Add linters to check my code for errors
Add lighthouse to check the quality of my webpage
Add webhint to  improve your site's accessibility
Add stylelint to avoid errors and enforce conventions in my styles.

### CODE REVIEW

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @29td, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights


## Required Changes ♻️
_Check the comments under the review._
- [x] Kindly add a description to your pull request. Click the three-dotted line in the image below to add and edit your description. Descriptions provide information about the changes you introduce to a particular branch.
image

- [x] Kindly add linters to your project and ensure the linters pass. Linters help us to adhere to industry standards when writing code. [Refer to this link](https://github.com/microverseinc/linters-config/tree/master/html-css)

- [x] Add information about your project and social media and GitHub handles to your readme file to make it more descriptive
image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -25,32 +25,32 @@
...
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- GitHub: [@29td](https://github.com/githubhandle)
+- Twitter: [@NonhlanhlaMnde6](https://twitter.com/twitterhandle)
 - LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
 
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```
Comment (Kossi-stack) [inline README.md]: - [x] Kindly remove Author 2 since you are alone on this project.

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
**Hi @29td,**

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

**To Highlight**

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ Commit messages is meaningful.

**Required Changes ♻️**

 - [x] Please make sure to use [this readme template](https://github.com/microverseinc/readme-template/blob/master/README.md) and please be sure to edit all of the sections to better suit your project.

 _Check the comments under the review for more changes_.

**Optional suggestions**
 - [x] Kindly edit the PR's title to something more descriptive. it can be "Hello Microverse Project PR or Hello Microverse PR".
 - [x] Kindly add test.md and node_modules to gitignore file.
 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md [hunk 2]
```diff
@@ -1,56 +1,48 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
```
Comment (Ismailco) [inline README.md]: - [x] Linter should be set up correctly for this repository, kindly add the [linters config files using this repo](https://github.com/microverseinc/linters-config)

Comment (Ismailco) [inline README.md]: - [x] This Pull Request should have a summary describing introduced changes in more detail, kindly edit the description so it meets this requirement

Comment (Ismailco) [inline README.md]: - [x] Branch should have a name of a feature or action e.g. ```created_sidebar``` or ```added_comments_section```, you can skip this requirement in this PR but please in future consider implementing this

Comment (Ismailco) [inline README.md]: - [x] There should be a ```.gitignore``` file listing test.md file and node_modules directory in the repo, please add a this into the gitignore file

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @29td,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

✔️ Used Github Flow
✔️ Work documented professionally

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by 29td** [comment]
Hi @Ismailco
Thank you for the feedback and I will make the changes and hope to do the
same in future as well. Will this feedback stop me from continuing with the
next project? Am I granted the opportunity to continue with the next project

**Comment by Ismailco** [comment]
Hi @29td, if you do the changes and fix the conflicts in this branch then yes you can request another review, and get approved

**Review by Davidosky007 — COMMENTED** [review]
Hello @29td  👋,

_This is an additional review as a result of the second opinion request._

* Kindly setup your linters properly by following the steps in this link below 👇 as your linter checks are still missing and try to work on the OPTIONAL cahnges in your spare time 🙏 
https://github.com/microverseinc/linters-config/tree/master/html-css 


As this was the only reason that your project was not approved it will be approved as soon as possible.
Please request for another Code review in your dashboard 
Keep rocking 🚀!

**Pull request closed without merging**